src/components/BasicHead.vue

```diff
@@ -244,6 +241,20 @@ export default {
   },
   
   methods: {
+    defaults () {
+      let features = ['print', 'bookmark', 'collect']
+      this.featuresMenu = []
+      for(let feature of features) {
+        if(this.tool[feature]) {
+          this.featuresMenu.push(Object.assign({}, 
+            this.tool[feature], 
+            {title: feature, click: this[this.tool[feature].click]}))
```

Comment:
replace this approach with the way the add button is implemented - all tool buttons need to work the same way or we will lose maintainability

Reply:
Done